### ISSUE TITLE
ERC20 address verification from bountyblok

### ISSUE THREAD

**Issue by jackiechao1** [BlockPILabs/testnets#336]
Provider name: Bountyblok
Discord Tag: jackie._.c
ERC20: 0xf1B75e876ED49eEb522E4aD1b5f8C88A0361A727

**Comment by Mummyhap** [comment]
Discord role verified
Please double check this ERC20 address because it is not the original one in the application form,
0xf1B75e876ED49eEb522E4aD1b5f8C88A0361A727

**Comment by jackiechao1** [comment]
Yes, had to use a different ERC20 address as we lost access to the original.

**Comment by Mummyhap** [comment]
Cool. The rewards will be delivered by the end of this month.
